app/experimenter/experiments/email.py

```diff
@@ -14,96 +14,75 @@ def send_intent_to_ship_email(experiment_id):
     # Because that's how it's done in Experiment.population (property)
     percent_of_population = f"{float(experiment.population_percent):g}%"
 
-    content = render_to_string(
-        "experiments/emails/intent_to_ship.txt",
+    format_and_send_html_email(
+        experiment,
+        "experiments/emails/intent_to_ship.html",
         {
             "experiment": experiment,
             "bug_url": bug_url,
             "percent_of_population": percent_of_population,
             "locales": [str(l) for l in experiment.locales.all()],
             "countries": [str(c) for c in experiment.countries.all()],
         },
-    )
-    # Strip extra newlines from autoescape tag
-    content = content.strip() + "\n"
-
-    channel = experiment.firefox_channel
-    email = EmailMessage(
-        Experiment.INTENT_TO_SHIP_EMAIL_SUBJECT.format(
-            name=experiment.name,
-            version=experiment.format_firefox_versions,
-            channel=channel,
-        ),
-        content,
-        settings.EMAIL_SENDER,
-        [settings.EMAIL_RELEASE_DRIVERS],
-        cc=[experiment.owner.email],
-    )
-    email.send(fail_silently=False)
-
-    ExperimentEmail.objects.create(
-        experiment=experiment, type=Experiment.INTENT_TO_SHIP_EMAIL_LABEL
+        Experiment.INTENT_TO_SHIP_EMAIL_SUBJECT,
+        Experiment.INTENT_TO_SHIP_EMAIL_LABEL,
+        cc_recipients=[settings.EMAIL_RELEASE_DRIVERS],
```

Comment:
Oh I see so emails always go to the owner and subscribers, and then intent to ship just also goes to the release drivers.  Yeah that seems sensible.

Reply:
yea that's it!